src/components/Items/ProfileContact.js

```diff
@@ -18,10 +19,14 @@ const ProfileContact = ({
   item,
   ImageComponent = null,
   disabled = false,
-  from
+  isYou
 }) => {
   const {anonProfileId, signedProfileId} = useProfileHook();
   const handleYouText = () => {
+    if (isYou) {
+      return '(You)';
+    }
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `isYou` prop has been introduced to determine if the current user is the one being rendered. This seems like a useful addition for customization based on user identity. However, there might be redundancy in the `handleYouText` function. If `isYou` is true, it will return '(You)', and then it checks again if the `anonProfileId` or `signedProfileId` matches the `item.user_id`, which could also return '(You)'. If `isYou` is always accurate, the second check might be unnecessary.

```diff
-  if (anonProfileId === item?.user_id || signedProfileId === item?.user_id) {
-    return '(You)';
-  }
+  if (!isYou && (anonProfileId === item?.user_id || signedProfileId === item?.user_id)) {
+    return '(You)';
+  }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->